build.sbt

```diff
@@ -26,22 +26,25 @@ checksums in update := Nil
 
 libraryDependencies ++= Seq(
 
-  "org.apache.spark" %% "spark-core" % testSparkVersion.value % "provided" force(),
```

Comment:
It seems to lose the correct version information.
For example, `org.apache.httpcomponents:httpclient` was `4.5.2`. (This is the version in Spark, too). Now, it's `4.4`.